### DISCUSSION TITLE
New Divider Text Color

### DISCUSSION THREAD

**Discussion by NikarashiHatsu** [saadeghi/daisyui#2509] [Ideas / feature requests]
<img width="925" alt="Screenshot 2023-11-13 at 14 00 27" src="https://github.com/saadeghi/daisyui/assets/43662951/4ad11a62-7b15-48c4-91bf-fa1894744f32">

I do really think that it is best to comply the default text color corresponding to each of the dividers. I need opinions for this idea.

**Comment by saadeghi** [comment]
Setting the text color like this may create accessibility issues. 
Because primary color for example can be any color (either from daisyUI built-in themes or custom themes). It may not have necessarily a good contrast with the base-100 color and if that's the case the text would be hard to read.

`primary` color is expected to have a readable content with `primary-content` color.   
And `base-100` color is expected to have a readable contrast with `base-content` color.

You can manually set a text color to the divider if you need the result in the screenshot. Just make sure the text is readable in your themes.

**Reply by NikarashiHatsu** [reply]
I see, that's a solid reason.
Thank you for your reply, I appreciate it!
